_clients/data-prepper/migrate-open-distro.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: Migrating from Open Distro Data Prepper
+parent: Data Prepper
+nav_order: 2
```

Comment:
Is this the left-hand menu order?

I think this is probably a fairly low-priority page as many users have already moved to OpenSearch Data Prepper. I'd suggest having it toward the bottom.